doc/R-vignette/igraph_extended_tutorial_R_vignette_english.Rmd

```diff
@@ -0,0 +1,682 @@
+
+```{=html}
+<!---
+I take this introducction from the R tutorial in https://igraph.org/r/html/latest/aaa-igraph-package.html
+-->
+```
+
+## Creating a graph
+
+The simplest way to create a graph is the `graph` constructor. To make an empty graph:
+
+```{r}
+library("igraph")
+
+g <- make_empty_graph()
+```
+
+To make a graph with 10 nodes (numbered `1` to `10`) and two edges connecting nodes `1-2` and `1-5`:
+
+```{r}
+g <- make_graph(edges = c(1,2, 1,5), n=10, directed = FALSE)
+```
+
+We can print the graph to get a summary of its nodes and edges:
+
+```{r echo = TRUE}
+g
+```
+
+This means: **U** ndirected graph with **10** vertices and **2** edges, with the exact edges listed out. If the graph has a [name] attribute, it is printed as well.
+
+Note
+
+`summary` does not list the edges, which is convenient for large graphs with millions of edges:
+
+```{r echo = TRUE}
+summary(g)
+```
+
+`make_graph` can create some notable graphs. For example you can create a Frucht graph, who is the smallest cubical graph whose automorphism group consists only of the identity element. It has 12 vertices and 18 edges:
+
+```{r echo = TRUE}
+g <- make_graph('Frucht')
+```
+
+To visualize a graph you can use `plot`:
+
+```{r}
+plot(g)
+```
+
+You can also include literals here. In this case, the first term of the formula has to start with a '~' character, just like regular formulae in R. The expressions consist of vertex names and edge operators. An edge operator is a sequence of ‘-’ and ‘+’ characters, the former is for the edges and the latter is used for arrow heads. The edges can be arbitrarily long, ie. you may use as many ‘-’ characters to “draw” them as you like. If all edge operators consist of only ‘-’ characters then the graph will be undirected, whereas a single ‘+’ character implies a directed graph:
+
+```{r echo = TRUE}
+g <- make_graph( ~ A-B, C-D, E-F, G-H, I, J, K)
+g
+```
+
+## Vertex and edge IDs
+
+Vertices and edges have numerical vertex IDs in igraph. Vertex IDs are always consecutive and they start with one. I.e. for a graph with n vertices the vertex IDs are between 1 and n. If some operation changes the number of vertices in the graphs, e.g. a subgraph is created via induced_subgraph, then the vertices are renumbered to satisfy this criteria.
+
+The same is true for the edges as well, edge IDs are always between one and m, the total number of edges in the graph.
+
+It is often desirable to follow vertices along a number of graph operations, and vertex IDs don't allow this because of the renumbering. The solution is to assign attributes to the vertices. These are kept by all operations, if possible.
+
+
+## Adding/deleting vertices and edges
+
+Let's start from the empty graph again. To add vertices to an existing graph, use `add_vertices`:
+
+```{r}
+g <- make_empty_graph()
```

Comment:
@adriandiazlab Let's make a non-empty graph, e.g. again Zachary's karate club, and write a sentence that says: "After a graph is constructed, you can add vertices to it using the following function:"